R/warnings.R

```diff
@@ -60,7 +60,7 @@ warn_if_vars_exist <- function(dataset, vars) {
 #'
 #' ## Issues a warning
 #' warn_if_invalid_dtc(dtc = "2021-04-06T-:30:30")
-warn_if_invalid_dtc <- function(dtc, is_valid = is_valid_dtc(dtc)) {
+warn_if_invalid_dtc <- function(dtc, is_valid = is.POSIXlt(dtc)) {
```

Comment:
I think this does not work as expected. E.g., `warn_if_invalid_dtc("2020")` issues a warning now.